src/test/regress/expected/subquery_in_where.out

```diff
@@ -24,10 +22,9 @@ DEBUG:  Plan XXX query after replacing subqueries and CTEs: SELECT count(*) AS c
    101
 (1 row)
 
-SET citus.enable_cte_inlining TO true;
```

Comment:
This test specifically expects non-materialized CTEs. Can we undo `MATERIALIZED`?